helm-chart/renku/templates/gateway/configmap.yaml

```diff
@@ -95,4 +95,8 @@ data:
       prometheus:
         enabled: {{ .Values.gateway.metrics.enabled }}
         port: {{ .Values.gateway.metrics.port }}
+    redirects:
```

Comment:
suggestion: the gateway does and in the future may do other redirects. I would call this `gitlabRedirects` or something that makes it clearer.

Reply:
That works Sekhar.

Reply:
Fixed in 7ce444aa619ba921c75ab4219d2ed1ee8f256ab7